grunt_tasks/package.js

```diff
@@ -38,8 +38,8 @@ module.exports = function (grunt) {
 
                         var fname = 'girder-' + grunt.config.get('pkg').version +
                             '.tar.gz';
-                        var stat = fs.statSync(fname);
-                        if (!err && stat.isFile() && stat.size > 0) {
+                        var stat = fs.existsSync(fname) && fs.statSync(fname);
+                        if (!err && stat && stat.isFile() && stat.size > 0) {
```

Comment:
I'm not too familiar with this part.  Are these changes related to the build process?